integrations/wiz/.port/spec.yaml

```diff
@@ -0,0 +1,30 @@
+type: wiz
+description: Wiz integration for Port Ocean
+icon: Alert
+features:
+  - type: exporter
+    section: Incident Management
+    resources:
+      - kind: issue
+      - kind: project
+      - kind: control
+      - kind: serviceTicket
+configurations:
+  - name: wizClientId
+    required: true
+    type: string
+    sensitive: true
+  - name: wizClientSecret
+    required: true
+    type: string
+    sensitive: true
+  - name: wizApiUrl
+    type: string
+    required: true
+  - name: wizTokenUrl
+    type: string
+    required: true
+  - name: wizWebhookToken
```

Comment:
rename here in the config.yaml and the main to wizWebhookVerificationToken

Reply:
alright